src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+# VMware Tanzu Edge Solution Architecture
+
+Edge computing is gaining a lot of momentum for a variety of reasons that range from avoidance of latency, data volume, and bandwidth considerations to need for autonomous disconnected operation, privacy and security. These reasons are pushing businesses across all industries to run more and more applications at edge locations. Traditionally, these applications were run in virtual machines, but with emergence of Kubernetes and microservices architectures, more applications are being containerized and there is a need for running a cloud native compute stack at the edge.
```

Comment:
... to the need for autonomous ... operations, privacy, and ... Traditionally, these applications are ... run on VMs, ... but with